### PR TITLE
feat: added uninstall command

### PR DESCRIPTION
**Pull request by azar-intelops**
Hi @pratikjagrut ,

I have added uninstall command. Please let me know if anything need to be added :)

### CODE REVIEW

**Comment by pratikjagrut** [comment]
@azar-intelops Please add better description take a look at description from install command's PR.

File: cmd/tarianctl/cmd/uninstall/uninstall.go [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+		g.Go(func() error {
+			return f()
+		})
+	}
+	err = g.Wait()
+	if err != nil {
+		return fmt.Errorf("uninstall: failed to uninstall: %w", err)
+	}
+
+	if c.onlyAgents {
```
Comment (pratikjagrut) [inline cmd/tarianctl/cmd/uninstall/uninstall.go:121]: I don't think uninstalling onlyAgents would work. Can you check it once and let me know the results?

**Review by pratikjagrut — CHANGES_REQUESTED** [review]

File: cmd/tarianctl/cmd/uninstall/uninstall.go (hunk 1 above)
Comment (MrAzharuddin) [reply cmd/tarianctl/cmd/uninstall/uninstall.go:121]: Hi @pratikjagrut , I have added unit test cases for that, May be need some more information to investigate

**Comment by devopstoday11** [comment]
@pratikjagrut @azar-intelops , please resolve this PR and close it ASAP.

**Pull request closed without merging**